pep-0308.txt

```diff
@@ -5,318 +5,332 @@ Last-Modified: $Date$
-    favor nor against this proposal; it is up to the community to
-    decide.  If the community can't decide, the BDFL will reject the
-    PEP.
-
-    After unprecedented community response (very good arguments were
-    made both pro and con) this PEP has been revised with the help of
-    Raymond Hettinger.  Without going through a complete revision
-    history, the main changes are a different proposed syntax, an
-    overview of proposed alternatives, the state of the curent
-    discussion, and a discussion of short-circuit behavior.
-
-    Following the discussion, a vote was held.  While there was an overall
-    interest in having some form of if-then-else expressions, no one
-    format was able to draw majority support.  Accordingly, the PEP was
-    rejected due to the lack of an overwhelming majority for change.
-    Also, a Python design principle has been to prefer the status quo
-    whenever there are doubts about which path to take.
+=======================================================================
+
+Requests for an if-then-else ("ternary") expression keep coming up
+on comp.lang.python.  This PEP contains a concrete proposal of a
+fairly Pythonic syntax.  This is the community's one chance: if
+this PEP is approved with a clear majority, it will be implemented
+in Python 2.4.  If not, the PEP will be augmented with a summary
+of the reasons for rejection and the subject better not come up
+again.  While the BDFL is co-author of this PEP, he is neither in
+favor nor against this proposal; it is up to the community to
+decide.  If the community can't decide, the BDFL will reject the
+PEP.
+
+After unprecedented community response (very good arguments were
+made both pro and con) this PEP has been revised with the help of
+Raymond Hettinger.  Without going through a complete revision
+history, the main changes are a different proposed syntax, an
+overview of proposed alternatives, the state of the curent
+discussion, and a discussion of short-circuit behavior.
+
+Following the discussion, a vote was held.  While there was an overall
+interest in having some form of if-then-else expressions, no one
+format was able to draw majority support.  Accordingly, the PEP was
+rejected due to the lack of an overwhelming majority for change.
+Also, a Python design principle has been to prefer the status quo
+whenever there are doubts about which path to take.
 
 
 Proposal
+========
 
-    The proposed syntax is as follows:
+The proposed syntax is as follows::
 
-        (if <condition>: <expression1> else: <expression2>)
+    (if <condition>: <expression1> else: <expression2>)
 
-    Note that the enclosing parentheses are not optional.
+Note that the enclosing parentheses are not optional.
 
-    The resulting expression is evaluated like this:
+The resulting expression is evaluated like this:
 
-    - First, <condition> is evaluated.
+- First, <condition> is evaluated.
```

Comment:
Brackets don't need to be escaped?

Reply:
@merwok Yes, `docutils` will do it.

Reply:
I didn’t mean HTML-escaped, but reST-escaped, since angle brackets have a syntactic role inside `ref` markup.  But it seems like this usage is ok!